### ISSUE TITLE
Normalisation options in Seurat Object

### ISSUE THREAD

**Issue by LeaLe88** [satijalab/seurat#8365]
Good morning,

I am currently preparing a deconvolution, for which I am using scRNA & bulk RNA Seq data.
It is recommended that both should be represented in same normalisation space.

Looking at the normalization.method function in Seurat I see the following options:
![image](https://github.com/satijalab/seurat/assets/71077063/e4ad7dfc-ac01-4942-a678-d4875257fab1)

I would like to obtain a length normalised method (like RPKM, FPKM, TPM) - how would that be possible here?

Thank you very much!

**Comment by longmanz** [comment]
Hi, 
In Seurat we do not support length-based normalization methods, because length information is not stored. You might need to do it with alternative software + your original scRNA-seq data. 
Which deconvolution method are you using? Does it explicitly ask you to use the same length-based normalization method to your RNAseq and scRNAseq data?

**Comment by LeaLe88** [comment]
Sorry for the late reply, just to close this topic some comments:
I am using CIBERSORT. I did not find information in the CIBERSORT support regarding which normalisation method is best for scRNA however I read for bulkRNA seq it is best to use TPM counts and I read somewhere that it is good if both are in the same normalisation space.
However in the meantime I found this paper https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03016-6/figures/4 
It says e.g. for kidney it doesnt really matter if you use no normalisation, lognorm or TMM counts:

![image](https://github.com/satijalab/seurat/assets/71077063/5558fb40-9c47-4ee1-8728-d65bae1d6c9a)

So I am still a bit confused whether it should be in the same normalisation space or not. Any further opinions appreciated.
